Bottom: 87d3a4a0bb10a5b823561ea0b548d94a128acb79
Top:    945d210407a3963aa08a7802babea888093e4806
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-11 12:08:08 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
